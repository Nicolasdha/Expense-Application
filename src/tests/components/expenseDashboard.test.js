import React from 'react';
import { shallow } from 'enzyme';

import ExpenseDashboardPage from '../../components/expenseDashboard';


test('should set up expense dashboard page', () =>{
    const wrapper = shallow(<ExpenseDashboardPage />);

    expect(wrapper).toMatchSnapshot();
});
