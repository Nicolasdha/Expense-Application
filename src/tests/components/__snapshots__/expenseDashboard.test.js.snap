// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should set up expense dashboard page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpensesSummary) />
  <Connect(ExpenseList) />
</div>
`;
